src/components/playlist-modal/PlaylistModal.jsx

```diff
@@ -0,0 +1,126 @@
+import { useEffect, useState } from "react";
+import "./playlist-modal.css";
+import { useVideoAction } from "../../contexts";
+import { useNavigate } from "react-router-dom";
+import {
+  addNewPlaylist,
+  addVideoToPlaylist,
+  getAllPlaylists,
+  isVideoInPlaylist,
+  removeVideoFromPlaylist,
+} from "../../contexts/video-action-context/playlist-utils";
+import { MdPlaylistAddCheck, MdPlaylistAdd } from "react-icons/md";
+import { Toast } from "..";
```

Comment:
i guess import is not properly, check your path once

Reply:
thansk a lot shubham!!